### ISSUE TITLE
Askomics ontology

### ISSUE THREAD

**Issue by sletort** [askomics/askomics#281]
_askomics:entity_ is defined as a relation that is always true or does not exist.
`up:Protein askomics:entity "true"^^xsd:boolean ;`
It seems more relevant to describe it as a type :
`up:Protein rdf:type askomics:entity`

I have the same reflexion with _askomics:startPoint_, even if it's more understandable to use true/false. We never use false. And as startpoints are always and only entities, we could say that:
`up:Protein rdf:type askomics:startpoint` implies `up:Protein rdf:type askomics:entity`.
-> note that in practice I'm not sure that we do not write both:
`up:Protein rdf:type askomics:entity;
rdf:type askomics:startpoint ;
.`

I don't know who decide.
It will change some query behaviour.

**Comment by xgaia** [comment]
I agree to replace `up:Protein askomics:entity "true"^^xsd:boolean` with `up:Protein rdf:type askomics:entity`

For the `askomics;startpoints,` when users integrate a file, ha can choose if the entity is a start point

![image](https://user-images.githubusercontent.com/18330770/47221934-e0de3380-d3b5-11e8-9abd-0e5125aa9a62.png)

If not, entity will never be displayed in the start points page

![image](https://user-images.githubusercontent.com/18330770/47222038-1a16a380-d3b6-11e8-8ed3-ac8c7258184e.png)
